apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/RedigerFamilieforhold.tsx

```diff
@@ -139,13 +143,17 @@ export const RedigerFamilieforhold = ({ behandling, personopplysninger }: Props)
                       />
                     </Box>
                     <div>
-                      <Button icon={<XMarkIcon />} variant="tertiary" onClick={() => gjenlevendeListe.remove(index)} />
+                      <Button
+                        icon={<XMarkIcon title="Fjern gjenlevende" />}
```

Comment:
Burde ikke `title` være på knappen i stedet for ikonet? 
Mye av arealet tilknyttet knappen (eks padding) vil aldri trigge visning av tittel på `onhover` siden ikonet bare er en liten del av knappen

Reply:
TIL!